ui/src/kapacitor/apis/index.js

```diff
@@ -100,3 +100,18 @@ export const updateTask = async (
     throw error
   }
 }
+
+export const getLogStream = kapacitor =>
+  fetch(`${kapacitor.links.proxy}?path=/kapacitor/v1preview/logs`, {
+    method: 'GET',
```

Comment:
Will this have to be changed to /v1 (no preview)?

Reply:
You'd have to ask @desa @nathanielc or @goller